

t = 1

try:
    print(t/1)
    print(1/0)
except Exception as e:
    print(e)